var/spack/repos/builtin/packages/jags/package.py

```diff
@@ -0,0 +1,44 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Jags(AutotoolsPackage):
+    """JAGS is Just Another Gibbs Sampler.  It is a program for analysis of
+       Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC)
+       simulation  not wholly unlike BUGS"""
+
+    homepage = "http://mcmc-jags.sourceforge.net/"
+    url = "https://downloads.sourceforge.net/project/mcmc-jags/JAGS/4.x/Source/JAGS-4.2.0.tar.gz"
+
+    version('4.2.0', '9e521b3cfb23d3290a8c6bc0b79bf426')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    depends_on('netlib-lapack')
```

Comment:
So you're saying that any LAPACK provider should work, but in practice only `netlib-lapack` and `atlas` seem to work? In that case, I would use:
```python
depends_on('lapack')

# Only netlib-lapack and atlas work
conflicts('^intel-mkl')
conflicts('^intel-parallel-studio')
conflicts('^openblas')
conflicts('^veclibfort')
```